music_metadata_extractor/converter.py

```diff
@@ -0,0 +1,10 @@
+import logging
+from urllib import parse
+
+host = 'youtube.com'
```

Comment:
Could we move all configuration and constants to another place and import from there? It seems a little strange to define this here. Also, although this seems right, could we check with a couple more corner cases since there is such a wide variety of YT links

Reply:
Yes, I thought of that too. Let me make that change

Reply:
As for YT music link, I checked, there is no short URL or any other type for it like for YT there is `m.youtube`, `youtu.be`

Reply:
Cool 👍